### DISCUSSION TITLE
Recognition of white digits with red background fails

### DISCUSSION THREAD

**Discussion by Brakelmann** [jomjol/AI-on-the-edge-device#1508] [General] [answered]
**Describe the bug:**
Recognition of white digits with red background fails.
The recognition of the sixth digit, the first after the comma fails.

**Version:**
V 11.3.1

**Screenshots:**
![image](https://user-images.githubusercontent.com/32041481/206123893-3570b610-6d58-4ad6-8047-762c3e34b206.png)

![image](https://user-images.githubusercontent.com/32041481/206123998-8e968e14-44e4-4cb7-8cdb-b242aea4a4bc.png)

The are similar red digits in the training data: https://jomjol.github.io/neural-network-digital-counter-readout/ 
Maybe it is a problem to have mixed background colors (black & red)?
Is there any chance to get a better recognition rate?

Thanks in advance for any support.

**Comment by jomjol** [comment]
What tflite do you use? It should work if you use a newer tflite.

**Comment by Kreatinbulle** [comment, marked as answer]
I have the same watermeter and for me, it works perfectly with dig-class100-0140-s2. Please try that one

**Comment by Brakelmann** [comment]
I am using the standard file I guess, did not change the tflite.

![Bildschirmfoto 2022-12-07 um 19 00 56](https://user-images.githubusercontent.com/32041481/206260872-1e9feeaa-df0f-497c-b2c2-9829a3d91978.png)

I will try to find the mentioned "dig-class100-0140-s2" and use this one.

Thanks.

**Comment by Brakelmann** [comment]
Yep, that did the trick.
Downloaded the tflite from the latest git release, uploaded the file changed the config, rebooted and it works.
Just wondering why this tflite was not in the config folder.

Thanks guys.
